### DISCUSSION TITLE
Artisan settings missing after re-starting the app

### DISCUSSION THREAD

**Discussion by 3Chord** [artisan-roaster-scope/artisan#1255] [Q&A] [answered]
I have Artisan running on two Apple laptops. A 2013 using Catalina (OS 10.15.7). Artisan works perfectly every time.
 
And a 2015 laptop using Monterey (OS 12.6.8). On this laptop I can configure Artisan settings to match my other laptop - **BUT** - after I quit the app and then re-open all the settings are missing. Nothing has been saved. Artisan requires every setting to be re-programmed. I have un-installed and re-installed several times and that isn't the problem. I'm unable to think of anything that I'm doing incorrectly after several attempts. I follow the steps in the online guide each time. Really frustrating.

Has anyone else encountered this? What am I doing wrong? I want to use the newer laptop (more speed and better battery). If this is covered in another discussion please let me know. Many thanks.

**Comment by roasterdave** [comment, marked as answer]
What you are reporting is not the expected behavior.  As a first step, please try the following and report back your results.  

1. On the machine that works properly do you see the ON and START buttons?
2. On that machine save Artisan's settings to a file (menu Help>> Save settings).  
3. Take a copy of that file to the newer computer.
4. Open Artisan on the newer computer.  
5. Do you see the ON and START buttons?  
6. If yes, load the settings file into Artisan (Help>> Load settings).
7. Look around and confirm the settings you expect were loaded.
8. Close and then re-open Artisan.
9. Do you see the ON and START buttons?  
10. If yes, are the settings as you expect?

**Comment by 3Chord** [comment]
Hi Dave. 

Thanks for this information. I had been wondering how to get the Artisan info from the laptop where it works perfectly to the other laptop where it doesn’t - and now I know how. Will Artisan give me a menu to save the file to my desktop?

I will report back shortly.

David Wallen

**Reply by roasterdave** [reply]
Yes, Artisan will provide a normal file save dialog.

**Comment by 3Chord** [comment]
The results are encouraging. Here’s the update:

When I set up Artisan on my 2015 Macbook Pro I manually inserted all settings to match with my 2013 Macbook Pro. Obviously more work than simply importing the .alog file and loading it on the 2015 Artisan app. Why didn’t the setting adjustments I made save to the program and reload?

Also, if I make changes to settings in the current .alog (For example - where to autosave files on the new laptop, or changing the “Roaster Scope” title in properties, or the ranges for the color options) how do I tell Artisan to save those settings? Is it automatically saved? Do I save out the files and then reload (like your test procedure below)?

Many thanks!

**Reply by poundy** [reply]
You are confusing the whole situation. Please just follow the above sequence and report back. You mentioned .ALOG file, if that's what you transferred then you transferred a ROAST LOG not the ARTISAN SETTINGS file that you saved out which should have the .ASET extension.

**Comment by roasterdave** [comment]
`The results are encouraging. Here’s the update:`
What was the result?  What is the answer in step 10 above?  A **Yes** means Artisan is working  as intended on that computer.  

`Also, if I make changes to settings in the current .alog (For example - where to autosave files on the new laptop, or changing the “Roaster Scope” title in properties, or the ranges for the color options) how do I tell Artisan to save those settings? Is it automatically saved? `
Settings, like Autosave and colors, are automatically retained by Artisan when the program is closed.  Roast properties, like the Title, are specific to that roast, are saved in the roast's  .alog file and are not saved as a setting.  

`Do I save out the files and then reload (like your test procedure below)?`
That is not necessary for day-to-day operation.  Settings are retained by Artisan when the program is closed.   Writing settings to an .aset file allows for backup, transfer to another computer or troubleshooting by allowing us to see your configuration.

**Comment by 3Chord** [comment]
My apologies for the confusion here, as I followed the instructions to the letter, but wrote down .alog by mistake. My last post covered the results - which were “as expected”. Hopefully this clarifies the situation.
